packages/server/__snapshots__/cypress_spec.js

```diff
@@ -203,10 +203,10 @@ This machine sent the following parameters:
 {
   "osName": "darwin",
   "osVersion": "v1",
-  "browserName": "Electron",
-  "browserVersion": "59.1.2.3",
-  "specs": [
-    "cypress/e2e/app.cy.js"
+  "browserName": "Electron.... (Expected: Electron)",
+  "browserVersion": "59.1.2.3.... (Expected: 64)",
+  "differentSpecs": [
+    "cypress/integration/foo_spec.js"
   ]
```

Comment:
Curious how different specs come into play here. If this is starting the same parallel run, shouldn't this be a different set of specs ?

Reply:
Is possible that in the same way that the params can change, the `--spec` flag can be set in one machine, running a different set of specs?